index.html

```diff
@@ -5648,32 +5661,33 @@ <h3>Language and Base Direction</h3>
         </p>
 
         <p>
-Implementers utilizing JSON-LD are strongly urged to
-<a href="#extensibility">extend</a> the JSON-LD Context defining the
-internationalized <a>property</a> and use the Scoped Context feature of JSON-LD
-to alias the <code>@value</code>, <code>@language</code>, and
-<code>@direction</code> keywords to <code>value</code>, <code>lang</code>,
-and <code>dir</code>, respectively. An example of a JSON-LD Context snippet
-doing this is shown below.
+Multiple language value objects MAY be provided as an array value for the
+property:
         </p>
 
-        <pre class="example nohighlight" title="Specifying scoped aliasing for language information">
-"title": {
-  <span class="highlight">"@context": {"value": "@value", "lang": "@language", "dir": "@direction"}</span>,
-  "@id": "https://www.w3.org/2018/credentials/examples#title"
-}
+        <pre class="example nohighlight" title="Multiple language texts provided for title">
+"title": [
+  {
+    "@value": "<span class="highlight">HTML and CSS: Designing and Creating Websites</span>",
+    "@language": "<code>en</code>"
+  },
+  {
+    "@value": "<span class="highlight">HTML و CSS: تصميم و إنشاء مواقع الويب</span>",
```

Comment:
see above

Reply:
See response to "see above" :) -- https://github.com/w3c/vc-data-model/pull/1340/files#r1390255038

Reply:
Fixed as @aphillips suggested.